utils/io/file.go

```diff
@@ -84,28 +86,67 @@ func CopyDirectory(scrDir, dest string) error {
 	return nil
 }
 
-func Copy(srcFile, dstFile string) error {
-	out, err := os.Create(dstFile)
```

Comment:
changed the order, instead of creating the dest file first, we open the source file first. So that if source file doesn't exist, dest file won't be created